### ISSUE TITLE
Create a UI component for selecting an external vocabulary from CLAVAS

### ISSUE THREAD

**Issue by twagoo** [clarin-eric/component-registry-front-end#61]
https://trac.clarin.eu/wiki/CMDI%201.2/Vocabularies

**Comment by twagoo** [comment]
OpenSKOS 2 API has the following call:
[api/conceptscheme](http://145.100.58.79/clavas/public/api/conceptscheme?format=json) ([html rendering](http://145.100.58.79/clavas/public/api/conceptscheme?format=html))

Some more info provided by Olha from Meertens
> -- Eerste kennismaken: 
>
> http://145.100.58.79/ccr/public/api/institution?format=html
>
> en http://145.100.58.79/clavas/public/api/institution?format=html
>
> Vandaar kunt je browsen naar andere resources. 
>
> Bv.,  je kunt een lijst of een resource (institution, set, concept scheme, skos collection of concept) in rdf, json, jsonp of html aanvragen.
>
> -- voorbeelden in readme, sectie 5.7: 
>
> https://github.com/OpenSKOS/OpenSKOS/tree/developMeertens​ 
>
>
> -- Bugs/mankementen melden
>
> https://github.com/OpenSKOS/OpenSKOS/issues 
>
> Een voorbeeld van bugs/mankementen meelden voor meertens branch (zie #61):
>
> https://github.com/OpenSKOS/OpenSKOS/issues?q=is%3Aissue+is%3Aclosed​

**Comment by twagoo** [comment]
Back end has a proxy for the conceptscheme API at `/vocabulary/conceptscheme` as of clarin-eric/component-registry-rest@d5bcf54e26f0c363c9d9e35f1c297f1544cffc27

**Comment by twagoo** [comment]
Vocabulary items seem retrievable with a query in the form

`api/find-concepts?q=*:*&scheme={schemeUri}`

where `{schemeUri}` is obtained from the 'uri' field of a conceptScheme object.

__Query example__: http://145.100.58.79/clavas/public/api/find-concepts?q=*:*&scheme=http://openskos.meertens.knaw.nl/Organisations

for the scheme that was retrieved from `/api/conceptscheme` as
```json
{
"uri":"http://openskos.meertens.knaw.nl/Organisations",
"type":"http://www.w3.org/2004/02/skos/core#ConceptScheme",
"set":"http://mertens/knaw/dataset_361bee51-b554-427a-8055-5ad8490aecc4",
"dateSubmitted":"2016-10-11T15:44:25+00:00",
"creator":"http://localhost/clavas/public/api/users/c56fde4f-4f91-43a6-aa16-a3b3f2e3e51e",
"description@en":"Organization Vocabulary, bootstrapped by manual curation from the list of VLO organization names (http://www.clarin.eu/vlo).",
"title@en":"OpenSKOS-CLARIN Organizations","uuid":"3d859942-77e6-4c1a-9236-a574845291c7",
"tenant":{"uri":"http://mertens/knaw/formalorganization_c6c5b54d-4f7e-4516-a2ab-b5a1f0c25491"}
}
```

___Response:___
```
{
"response":
 {
  "numFound":32,
  "rows":5000,
  "start":0,
  "docs":[
    {"uri":"http://openskos.meertens.knaw.nl/Organisations/505d5864-8b5c-4766-ac32-f893a7663844","type":"http://www.w3.org/2004/02/skos/core#Concept","prefLabel@en":"D\u00e9partement FLE, Universit\u00e9 d\u0027Aix-Marseille","creator":{"uri":"http://localhost/clavas/public/api/users/c56fde4f-4f91-43a6-aa16-a3b3f2e3e51e"},"inScheme":{"uri":"http://openskos.meertens.knaw.nl/Organisations"},"status":"candidate","dateSubmitted":"2016-10-11T15:44:26+00:00","altLabel@en":"D\u00e9partement Fran\u00e7ais langue \u00e9trang\u00e8re / Universit\u00e9 d\u0027Aix-Marseille","uuid":"09512d11-b10c-4d64-9fcf-75ab0c1e9e41","set":"http://mertens/knaw/dataset_361bee51-b554-427a-8055-5ad8490aecc4","tenant":"http://mertens/knaw/formalorganization_c6c5b54d-4f7e-4516-a2ab-b5a1f0c25491"} 
...
```

**Comment by twagoo** [comment]
Additional information from Olha:
> http:​145.100.58.79/clavas/public/api/find-concepts?q=prefLabel:*&conceptScheme=http://openskos.meertens.knaw.nl/iso-639-3
other example:
>http://145.100.58.79/ccr/public/api/find-concepts?q=prefLabel:a*&conceptScheme=http://hdl.handle.net/11148/CCR_P-Metadata_deedbe7e-9a1d-4388-2857-ad0daaf06793

**Comment by twagoo** [comment]
Fixed as of 5c1add6

**Comment by twagoo** [comment]
Implemented (along with some related changes) as of ff77c3f
